lib/trino-record-decoder/src/test/java/io/trino/decoder/avro/TestAvroDecoder.java

```diff
@@ -1200,6 +1203,9 @@ public void testSupportedDataTypeValidation()
         singleColumnDecoder(new ArrayType(BigintType.BIGINT));
         singleColumnDecoder(VARCHAR_MAP_TYPE);
         singleColumnDecoder(DOUBLE_MAP_TYPE);
+        singleColumnDecoder(DateType.DATE);
+        singleColumnDecoder(TimeType.TIME_MILLIS);
+        singleColumnDecoder(TimestampType.TIMESTAMP_MILLIS);
```

Comment:
These assertion only verifies whether the type is supported in decoder. I would recommend adding more test likes `testLongDecodedAsBigint()`

Reply:
Add more tests for avro decoder and schema converter.

Reply:
The schema converter is of trino-kafka, reverted.